.github/workflows/ci.yaml

```diff
@@ -15,6 +16,8 @@ env:
   OTP_VERSION: 26
   MIX_ENV: test
   NODE_VERSION: "16"
+  MANTAINERS: '["cdimonaco", "dottorblaster", "fabriziosestito", "rtorrero", "nelsonkopliku", "arbulu89","jagabomb","emaksy","jamie-suse"]'
```

Comment:
Could we have all of this logic in an additional file? Like the `pr_env.yaml`
At the end, as it is triggered by a label, so a different flow, to have it separated

Reply:
I wrote an update in the PR description but you probably wrote this earlier: wouldn't this force us to repeat many of the steps that are already performed in the regular run? I supposedly could use the cache between different CI yamls but if I can't add dependencies I have no way of knowing if the assets in the cache are updated to the current commit or not. Maybe I'm missing something?

Reply:
I guess we can also add our newcomer: 

janvhs

Reply:
In fact, you could remove fabrizio as well